### PR TITLE
Fixing property types in EntityDeclaration

### PR DESCRIPTION
**Pull request by msmakouz**
- [x] Release [spiral/validator](https://github.com/spiral/validator) with fixed filters namespace and remove `dev` version from composer.json

### CODE REVIEW

**Comment by roxblnfk** [comment]
What about unit tests?

**Comment by msmakouz** [comment]
> What about failed tests?

Failed tests related to PR in SF:
https://github.com/spiral/framework/pull/747

And wait for a new release here:
https://github.com/spiral/validator

File: src/Scaffolder/Declaration/AbstractEntityDeclaration.php [hunk 1]
```diff
@@ -78,7 +79,24 @@ protected function isNullableType(string $type): bool
...
+
+        $phpMapping = [
+            'int'   => ['primary', 'bigPrimary', 'integer', 'tinyInteger', 'smallInteger', 'bigInteger'],
+            'bool'  => ['boolean'],
+            'float' => ['double', 'float', 'decimal'],
+            \DateTimeImmutable::class => ['datetime', 'date', 'time', 'timestamp']
+        ];
+
+        foreach ($phpMapping as $phpType => $candidates) {
+            if (in_array($type, $candidates, true)) {
```
Comment (roxblnfk) [inline src/Scaffolder/Declaration/AbstractEntityDeclaration.php:94]:
```suggestion
            if (\in_array($type, $candidates, true)) {
```

File: src/Scaffolder/Declaration/AbstractEntityDeclaration.php [hunk 2]
```diff
@@ -78,7 +79,24 @@ protected function isNullableType(string $type): bool
...
     private function variableType(string $type): string
     {
-        return $this->isNullableType($type) ? (substr($type, 1) . '|null') : $type;
+        if ($this->isNullableType($type)) {
+            $type = \substr($type, 1);
+        }
+
+        $phpMapping = [
+            'int'   => ['primary', 'bigPrimary', 'integer', 'tinyInteger', 'smallInteger', 'bigInteger'],
+            'bool'  => ['boolean'],
```
Comment (roxblnfk) [inline src/Scaffolder/Declaration/AbstractEntityDeclaration.php:88]:
```suggestion
            'bool'  => ['boolean', 'bool'],
```
Is `bool` valid value?

**Review by roxblnfk — APPROVED** [review]

File: src/Scaffolder/Declaration/AbstractEntityDeclaration.php (hunk 1 above)
Comment (msmakouz) [reply src/Scaffolder/Declaration/AbstractEntityDeclaration.php]: Fixed.

File: src/Scaffolder/Declaration/AbstractEntityDeclaration.php (hunk 2 above)
Comment (msmakouz) [reply src/Scaffolder/Declaration/AbstractEntityDeclaration.php]: Valid, thanks, added aliases:
https://github.com/cycle/database/blob/2.x/src/Schema/AbstractColumn.php#L210

**Review by butschster — APPROVED** [review]

**Review by roxblnfk — APPROVED** [review]

**Pull request merged**